internal/provisioner/utility/utility_group.go

```diff
@@ -241,6 +241,14 @@ func (group utilityGroup) ProvisionUtilityGroup() error {
 	for _, utility := range group.utilities {
 		logger.Infof("Provisioning utility %s\n", utility.Name())
 
+		if group.cluster.ActualUtilityVersion(utility.Name()).Chart == "unmanaged" {
```

Comment:
I think we probably want to check that the actual version and the desired version are unmanaged right? I know it's unlikely that we would want to go back the provisioner-managed, but this check wouldn't allow it as is.

Reply:
Thanks, Gabe. I changed it to consider the `desiredVersion` in the if statement.